### PR TITLE
Only setState if component if fully mounted

### PR DESCRIPTION
**Pull request by mgpGit**
fixes https://github.com/tajo/react-range/issues/117

### CODE REVIEW

**Comment by tajo** [comment]
Was it causing some error? Could it be captured by a test?

**Comment by tajo** [comment]
I would like to track down the root cause. Ideally, `getOffsets` should never get called unmounted component. I wonder if `onResize` is somehow called in your scenario?

Could you try to put together a small repro?

File: src/Range.tsx [hunk 1]
```diff
@@ -50,9 +50,11 @@ class Range extends React.Component<IProps> {
     markOffsets: []
   };
 
+
```
Comment (tajo) [inline src/Range.tsx:53]: please remove all these extra new lines

**Comment by tajo** [comment]
This looks good and makes sense.

File: src/Range.tsx (hunk 1 above)
Comment (mgpGit) [reply src/Range.tsx]: done!

**Comment by mgpGit** [comment]
@tajo - I removed the new lines. If everything looks good is this ok to merge the PR?

**Pull request merged**